test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -1780,7 +1780,8 @@ def convert_to_dictlist(self, l, attribs=None, mergelines=True, id=None):
                 m = self.pbsobjattrval_re.match(l)
                 if m:
                     attr = m.group('attribute')
-                    if attribs is None or attr in attribs:
+                    if (attribs is None or str(attr).lower() in attribs
+                            or attr in attribs):
```

Comment:
I'd either str() both instances of attr, or not str() them.  It should be a string coming from an re match, right?